torch_tvm/compiler.cpp

```diff
@@ -72,6 +72,9 @@ tvm::relay::Expr TVMCompiler::convertToRelay(
     auto v = tvm::relay::ConstantNode::make(x);
     return v;
   }
+  // TODO: This can be just NDArray of num elements of IntList.
```

Comment:
I believe that is true.  Curious, are you planning to use this fact for this PR? If not, maybe file an issue for improvement

Reply:
Nope. Not in this PR. Will file an issue for improvement.